includes/admin.form.inc

```diff
@@ -39,13 +39,74 @@ function islandora_newspaper_admin_settings_form(array $form, array &$form_state
-    '#size' => 30,
+  $form['islandora_newspaper_solr_options'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Solr settings'),
+    'islandora_newspaper_parent_issue_solr_field' => array(
+      '#type' => 'textfield',
+      '#title' => t('Parent Solr Field'),
+      '#description' => t("Solr field containing the parent issue's PID."),
+      '#default_value' => variable_get('islandora_newspaper_parent_issue_solr_field', 'RELS_EXT_isMemberOf_uri_ms'),
+      '#size' => 100,
+      '#autocomplete_path' => 'islandora_solr/autocomplete_luke',
+    ),
+    'islandora_newspaper_use_solr' => array(
+      '#type' => 'checkbox',
+      '#title' => t('Use Solr for Newspaper display'),
+      '#disabled' => (!module_exists('islandora_solr')),
+      '#description' => t('Use Solr to generate lists of issues for a newspaper object.'),
+      '#default_value' => (module_exists('islandora_solr') ? variable_get('islandora_newspaper_use_solr', FALSE) :
+                            FALSE),
+    ),
+    'islandora_newspaper_solr_wrapper' => array(
+      '#type' => 'container',
+      '#states' => array(
+        'visible' => array(
+          ':input[name="islandora_newspaper_use_solr"]' => array('checked' => TRUE),
+        ),
+      ),
+      'islandora_newspaper_solr_newspaper_parent_field' => array(
+        '#type' => 'textfield',
+        '#title' => t('Newspaper issue parent field'),
+        '#description' => t('Solr field that contains the parent newspaper of the newspaper issue.'),
+        '#size' => 100,
+        '#default_value' => variable_get('islandora_newspaper_solr_newspaper_parent_field', 'RELS_EXT_isMemberOf_uri_ms'),
+        '#autocomplete_path' => 'islandora_solr/autocomplete_luke',
+        '#states' => array(
+          'required' => array(
+            ':input[name="islandora_newspaper_use_solr"]' => array('checked' => TRUE),
+          ),
+        ),
+      ),
+      'islandora_newspaper_solr_date_field' => array(
+        '#type' => 'textfield',
+        '#title' => t('Issued date field'),
+        '#description' => t('Solr field that contains the date issued of the newspaper issue.'),
+        '#size' => 100,
+        '#default_value' => variable_get('islandora_newspaper_solr_date_field', 'RELS_EXT_dateIssued_literal_ms'),
+        '#autocomplete_path' => 'islandora_solr/autocomplete_luke',
+        '#states' => array(
+          'required' => array(
+            ':input[name="islandora_newspaper_use_solr"]' => array('checked' => TRUE),
+          ),
+        ),
+      ),
+      'islandora_newspaper_solr_sequence_field' => array(
+        '#type' => 'textfield',
+        '#title' => t('Sequence field'),
+        '#description' => t('Solr field that contains the sequence number of the issues in a newspaper.'),
+        '#size' => 100,
+        '#default_value' => variable_get('islandora_newspaper_solr_sequence_field', 'RELS_EXT_isSequenceNumber_literal_ms'),
```

Comment:
A bit of the same as previous, maybe better to default to something that can be sorted on via Solr (single valued)

Reply:
Same answer around sorting, depends on non-standard Solr config. Could be a new feature to add in later.